ci/lint/06_script.sh

```diff
@@ -14,18 +14,9 @@ if [ -n "$CIRRUS_PR" ]; then
     echo "Error: The top commit must be a merge commit, usually the remote 'pull/${PR_NUMBER}/merge' branch."
     false
   fi
-else
-  # Otherwise, assume that a merge commit exists. This merge commit is assumed
-  # to be the base, after which linting will be done. If the merge commit is
-  # HEAD, the range will be empty.
-  COMMIT_RANGE="$( git rev-list --max-count=1 --merges HEAD )..HEAD"
 fi
 export COMMIT_RANGE
```

Comment:
I think we can delete this line and then do the following on line 12
`export COMMIT_RANGE="HEAD~..HEAD"`

since now this export sometimes is empty